### ISSUE TITLE
[Feature Request] Add ability to add own commands to toolbar

### ISSUE THREAD

**Issue by Lonli-Lokli** [ForkIssues/TrackerWin#511]
Add ability to add own commands to toolbar:

eg if I want to rebase my branch on top of develop I have to do right clicks and clicking on buttons.
It will be much easier to create most used custom actions on toolbar by myself.

**Comment by DanPristupov** [comment]
No, this is too complicated. Every button must have an icon, etc. I doubt we will ever have time for that.

**Comment by Lonli-Lokli** [comment]
@DanPristupov  What about moving custom actions to the toolbar? Eg I can customize with git action but would like to use it on toolbar (with any default icon).
